data/mods/Magiclysm/effects/effects.json

```diff
@@ -163,5 +163,15 @@
     "rating": "good",
     "removes_effects": [ "eagles_sight", "cats_grace", "ogres_strength" ],
     "base_mods": { "int_mod": [ 4 ] }
+  },
+  {
+    "type": "effect_type",
+    "id": "feather_fall",
+    "name": [ "Feather Falling" ],
+    "desc": [ "You can fall long distances but are weakened!" ],
+    "apply_message": "Your body is lighter but weaker!",
+    "remove_message": "Your body returns to normal.",
+    "rating": "good",
+    "base_mods": { "dex_mod": [ 50 ], "str_mod": [ -4 ] }
```

Comment:
First let me apologize for all the guessing I'm about to do. Maybe just a typo.

Is 50 dex required to prevent fall damage? Seems like this could be abused with such a short cast time and mana cost. My first thought was archery - though I'll be honest, I don't know for sure how dex affects firing/bow damage, but I'm pretty sure its part of the calculation. Are there other activities that could greatly benefit from a quick burst of dex? I don't know, but thought it was worth mentioning.

Reply:
Seems you thought it. Sorry.

Reply:
I did the best I could, there's nothing else that I'm aware of. roughly 50 is required to be okay after a 6 story fall from an apartment building, which is where I tested.

Reply:
I think that casting this spell just to get off a shot or two from a bow would be a little ridiculous to say the least, the player would only be cheating themselves and might as well just outright cheat, but I can't stop them. If there was some effect to prevent attacking I would stack that in there.